NDTensors/src/blocksparse/blocksparse.jl

```diff
@@ -23,7 +23,7 @@ function set_datatype(storagetype::Type{<:BlockSparse}, datatype::Type{<:Abstrac
   return BlockSparse{eltype(datatype),datatype,ndims(storagetype)}
 end
 
-function SetParameters.set_ndims(storagetype::Type{<:BlockSparse}, ndims::Int)
+function TypeParameterAccessors.set_ndims(storagetype::Type{<:BlockSparse}, ndims::Int)
```

Comment:
This kind of code could be updated with `TypeParametersAccessors`. Should I make the update here?

Reply:
Let's just leave it as-is.